.github/workflows/ci.yml

```
@@ -0,0 +1,69 @@
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.x"
+      
+      - name: Install Dependencies
+        run: npm install
+        working-directory: ${{ github.workspace }}
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)